solutions/arrays/noble-integer.js

```diff
@@ -0,0 +1,34 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arraySize=parseInt(readLine())
+let arr=[]
+for(let index=0;index<arraySize;index++)
+{
+	arr.push(parseInt(readLine()));
+}
+let answer=0;
+for(let index=0;index<arraySize;index++)
+{
+	let count=0;
+	for(let index1=0;index1<arraySize;index1++)

```

Comment:
@DevratAgarwal Lets give an optimal solution in `O(nlogn)` time, I think the students will learn time complexity by the time of arrays.